wallet/src/de/schildbach/wallet/ui/invite/OnboardFromInvite.kt

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright 2021 Dash Core Group
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package de.schildbach.wallet.ui.invite
+
+import android.app.Activity
+import android.content.Context
+import android.content.Intent
+import android.graphics.Typeface
+import android.os.Bundle
+import android.widget.TextView
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.content.ContextCompat
+import de.schildbach.wallet_test.R
+import kotlinx.android.synthetic.main.activity_accept_invite.continue_button
+import kotlinx.android.synthetic.main.activity_onboard_from_invite.*
+
+class OnboardFromInvite : AppCompatActivity() {
```

Comment:
Should this class have "Activity" at the end of its name?

Reply:
Done